samples/esb/esp_ptx/README.rst

```diff
@@ -68,7 +59,12 @@ FEM support
 Testing
 =======
 
-After programming the Transmitter sample on one of the development kits and the Receiver sample on the other kit, test them by performing the following steps:
+After programming the Transmitter sample on one of the development kits and the Receiver sample on the other kit, you can test their functionality.
+
+.. note::
+   Follow the steps from and include the :ref:`esb_prx` sample to test the Transmitter sample.
```

Comment:
This note doesn't seem like it is needed since the instructions are to do everything on both kits. Following the note literally would lead to doing each step twice.